tests/ad_integration_tests.yml

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: MIT
+---
+# To run this test properly AD (ad1) must be configured and it's properties
+# provided with the following variables:
+# ad_integration_realm
+# ad_integration_user
+# ad_integration_password
+# Inventory must contain the client Linux node and the ad AD Server node like:
+# all:
+#   hosts:
+#     client:
+#       ansible_connection: local
+#       ansible_host: 127.0.0.1
+#     ad:
+#       ansible_host: 10.192.1.1
+#       ad_fqdn: ad1.domain.com
+#       ansible_connection: winrm
+#       ansible_password: Secret123
+#       ansible_port: 5986
+#       ansible_user: Administrator
+#       ansible_winrm_server_cert_validation: ignore
+- name: Test integration with AD server
```

Comment:
Our default ansible-lint configuration marks all files matching `tests/tests_*.yml` as playbooks, `tests/setup-snapshot.yml` as a playbook, and all other `tests/*.yml` files as task files.  Other roles have a `tests/playbooks/` directory for "other" test playbooks.  So we'll either have add this to the ansible-lint configuration, or move this to `tests/playbooks/`